src/Components/MobileNav.js

```diff
@@ -0,0 +1,112 @@
+import { useNavigate } from "react-router-dom";
+import { BiHistory } from "react-icons/bi";
+import { MdVideoLibrary } from "react-icons/md";
+import {
+  AiOutlineClockCircle,
+  AiFillClockCircle,
+  AiFillHeart,
+  AiOutlineHeart,
+  AiFillHome
+} from "react-icons/ai";
+
+export const MobileNav = () => {
+  const navigate = useNavigate();
+
+  return (
+    <>
+      <div className="mobile-nav mobile">
+        <div className="flex-row justify-space-evenly">
+          <AiFillHome
+            className="react-icon mobile-icon"
+            onClick={() => navigate("/")}
```

Comment:
Link tag/ NavLink can be used to redirect between pages

Reply:
Is that like the best practice when compared to useNavigate?

Reply:
useNavigate is usually for a programatic routing, but Link / NavLinks are just like <a> tag of html provided by react router. Navlink also provides you the extra flexibility to identify active links

Reply:
Got it!